release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -50,6 +50,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-13-installation-and-upgrade-three-node"]
+==== Three-node cluster support
+
+Beginning with {product-title} 4.12, deploying a three-node cluster is supported on Amazon Web Services(AWs), Google Cloud Platform (GCP), Microsoft Azure, and VMware vSphere. This type of {product-title} cluster is a smaller, more resource efficient cluster, as it consists of zero compute machines and three control plane machines only.
```

Comment:
How about replacing `as it consists of zero compute machines and three control plane machines only` with `as it only consists of three control plane machines, which also act as compute machines`? Because it's not really **zero compute machines**, but the 3 machines acting as both compute and control plane machines. Thanks!

Reply:
+1, actually, three nodes have both "master" and "worker" roles.
```
NAME                                                          STATUS   ROLES                               AGE    VERSION           INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                                                        KERNEL-VERSION                 CONTAINER-RUNTIME
ci-op-rbqqvr7j-77520-5flbs-master-0.c.XXXXXXXXXXXX.internal   Ready    control-plane,master,worker   6h7m   v1.26.0+b923723   10.0.0.5      <none>        Red Hat Enterprise Linux CoreOS 413.86.202302201445-0 (Ootpa)   4.18.0-372.43.1.el8_6.x86_64   cri-o://1.26.1-6.rhaos4.13.git159cc9c.el8
ci-op-rbqqvr7j-77520-5flbs-master-1.c.XXXXXXXXXXXX.internal   Ready    control-plane,master,worker   6h7m   v1.26.0+b923723   10.0.0.4      <none>        Red Hat Enterprise Linux CoreOS 413.86.202302201445-0 (Ootpa)   4.18.0-372.43.1.el8_6.x86_64   cri-o://1.26.1-6.rhaos4.13.git159cc9c.el8
ci-op-rbqqvr7j-77520-5flbs-master-2.c.XXXXXXXXXXXX.internal   Ready    control-plane,master,worker   6h6m   v1.26.0+b923723   10.0.0.3      <none>        Red Hat Enterprise Linux CoreOS 413.86.202302201445-0 (Ootpa)   4.18.0-372.43.1.el8_6.x86_64   cri-o://1.26.1-6.rhaos4.13.git159cc9c.el8
````

Reply:
> How about replacing `as it consists of zero compute machines and three control plane machines only` with `as it only consists of three control plane machines, which also act as compute machines`?

@jianli-wei @jinyunma Excellent point. Updated accordingly. Thank you.